backend/src/db/migrations/20240520064127_add-integration-sync-status.ts

```diff
@@ -0,0 +1,43 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const hasIsSyncedColumn = await knex.schema.hasColumn(TableName.Integration, "isSynced");
+  const hasSyncMessageColumn = await knex.schema.hasColumn(TableName.Integration, "syncMessage");
+  const hasLastSyncJobId = await knex.schema.hasColumn(TableName.Integration, "lastSyncJobId");
+
+  await knex.schema.alterTable(TableName.Integration, (t) => {
+    if (!hasIsSyncedColumn) {
+      t.boolean("isSynced").nullable();
```

Comment:
I see this column has never used again. Is this expected?

Reply:
we're using this in the frontend. backend returns this property as part of getById of integration
<img width="819" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/5e50261c-b934-41ad-a044-c00e6f25768f">

Reply:
My bad. Didn't find any reference when i did search for it. My mistake.